src/datasource.ts

```diff
@@ -25,7 +25,7 @@ export class Datasource extends DataSourceWithBackend<InfinityQuery, InfinityOpt
   query(options: DataQueryRequest<InfinityQuery>): Observable<DataQueryResponse> {
     return new Observable<DataQueryResponse>((subscriber) => {
       let request = getUpdatedDataRequest(options, this.instanceSettings);
-      reportQuery(request?.targets || [], this.instanceSettings, this.meta, request?.app);
+      // reportQuery(request?.targets || [], this.instanceSettings, this.meta, request?.app);
```

Comment:
Why are we not removing it completely and just commenting it out?

Reply:
disabling this is just a temporary solution, once sriram is back in office we can discuss enabling this again but finding a way to make this use less resources.

i guess we could still remove it, but since its just temp i thought commenting would be fine, WDYT?

Reply:
Can you then create a github issue to fix this properly. I don't think we are going to be able to use rudderstack for this and we can rather rely on logs/metrics from instrumentation. But let's discuss when sriram is back.